### PR TITLE
update:  add categories

### LINKED ISSUE #2: added navbar
Here I added navbar, which is responsive.

### PR DESCRIPTION
**Pull request by Bhakti199**
**Here I added categories to Navbar : Men, Women and Kids #2**

Screenshot (1320)

### CODE REVIEW

File: src/Components/Categories/Categories.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import axios from "axios";
+import "./Categories.css";
+import { Link } from "react-router-dom";
+const Categories = () => {
+  const [categoryList, setCategoryList] = useState([]);
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await axios.get("/api/categories");
```
Comment (tushartiwari7) [inline src/Components/Categories/Categories.jsx]: you may directly destructure responseand get data like: 
```
const {data} = await axios.get("/api/categories");
```

File: src/Components/Categories/Categories.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        const listCategory = response.data.categories;
+        console.log(listCategory);
```
Comment (tushartiwari7) [inline src/Components/Categories/Categories.jsx]: remove console logs before pushing to production.

**Review by tushartiwari7 — APPROVED** [review]

File: src/Components/Categories/Categories.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { useEffect } from "react";
+import { useState } from "react";
```
Comment (JantuDeb) [inline src/Components/Categories/Categories.jsx]: You can import all these in one line 
```import React, {useEffect, useState} from "react"```

**Review by JantuDeb — APPROVED** [review]
Suggested a changes

File: src/Components/Categories/Categories.jsx (hunk 3 above)
Comment (Bhakti199) [reply src/Components/Categories/Categories.jsx]: okay, I will update it.

**Pull request merged**